developers-guide/developer-docs/spv-wallet/notifications.md

```diff
@@ -0,0 +1,154 @@
+# Notifications
+
+## Overview
+
+The notifications system has the following features:
+
+- allows clients to subscribe and unsubscribe their webhook URLs
+- subscriptions are persisted in the database, so clients do not need to re-subscribe after an spv-wallet restart.
+- all instances of spv-wallet observe the status of currently-registered webhooks
+- each client has its own queue (golang's buffered channel) so a malfunctioning client does not block other clients
+- retries and timed bans on failure are implemented

```

Comment:
I've added the values/params what are defined for retries and ban logic. 
But how it works - it's pretty obvious/self-descriptive - I guess... so I haven't put it. But it would be like this:

1 The app tries to send a request 
2 On Failure waits a "delay" and tries once again [<retries> times]
3 If all retries have failed it is marked as banned for certain period of time 
4 After that time the app goes to point 1 and start to try :D 

If you want me to put something like this into the documentation - let me know.

Reply:
it's ok for me 👍